apps/studio/components/interfaces/Storage/VectorBuckets/CreateVectorBucketDialog.tsx

```diff
@@ -147,7 +153,7 @@ export const CreateVectorBucketDialog = () => {
                     name="name"
                     label="Bucket name"
                     labelOptional="Cannot be changed after creation"
-                    description="Must be between 3–63 characters. Valid characters are a-z, 0-9, hyphens (-), and periods (.). A target schema will be created that matches this name."
+                    description="A target schema will be created that matches this name."
```

Comment:
It’s a bit of a mouthful with the field validation spelt out. I know it’s (on paper) bad practice to hide the requirements, but they are so obvious that I want to sprinkle them in as needed.